public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -619,10 +620,20 @@ public ConceptAnalysis apply(ConceptAnalysis ca) {
                         }else {
                                 unitSeperateAnalysis.add(aa);
                         }
-
                     }
                     isMeasurement = true;
                     conceptAnalysis.setMeasurementValueAgeAnalysis(unitSeperateAnalysis.stream().map(TO_CLIENT_ANALYSIS).collect(Collectors.toList()));
+                }else if(analysisId == MEASUREMENT_GENDER_UNIT_ANALYSIS_ID){
+                    HashMap<String,List<AchillesResult>> results = seperateUnitResults(aa);
```

Comment:
nit: Better to use the more generic `Map` on the lhs, e.g. like you did with `List` below. HashMap is an irrelevant implementation detail.